### PR TITLE
Design: divide things into sections, add color

### PR DESCRIPTION
**Pull request by gabebw**
- Introduce our Trello board's background color as the `$accent-color` #tbt
- Use a two-column layout on the home page, with the recommendations feed taking up most of the space and recommended people to follow on the right.
- Add more whitespace all over.

---

Before:

youshouldtotally 2016-01-24 18-32-17

---

After:

youshouldtotally 2016-01-24 18-33-47

### CODE REVIEW

**Comment by gabebw** [comment]
@edwardloveall

File: app/assets/stylesheets/components/_header.scss [hunk 1]
```diff
@@ -1,9 +1,15 @@
 header {
-  background-color: $light-gray;
+  background-color: $accent-color;
+  color: white;
```
Comment (adarsh) [inline app/assets/stylesheets/components/_header.scss]: Should be a constant. Hound would catch this - should we set it up?

File: app/views/homes/show.html.erb [hunk 2]
```diff
@@ -1,25 +1,31 @@
-<%= simple_form_for TelevisionShow.new do |form| %>
-  <%= form.input :title %>
-  <%= form.button :submit %>
-<% end %>
+<main>
+  <h2>Recommendations</h2>
+  <%= simple_form_for TelevisionShow.new do |form| %>
+    <%= form.input :title, label: false, placeholder: "The Wire" %>
```
Comment (adarsh) [inline app/views/homes/show.html.erb]: I feel like `The Wire` is a serious default. What about something more frivolous?

Also, this should be a constant, right?

File: app/views/homes/show.html.erb [hunk 3]
```diff
@@ -1,25 +1,31 @@
...
 
-<h2>Recommendations</h2>
-<ul>
-  <% @recommended_items.each do |recommended_item| %>
-    <li>
-      <%= recommended_item.television_show_title %>
-      &mdash; <%= link_to recommended_item.username, recommended_item.user %>
-    </li>
+<aside>
+  <% if current_user.following.empty? %>
```
Comment (adarsh) [inline app/views/homes/show.html.erb:21]: Maybe this should be in a different PR but what if this was just a generic "Recommended People To Follow" list, which was somehow _more prominent_ when there is no one? Or are those two separate things?

**Comment by adarsh** [comment]
All good. Is this responsive? If not, probably separate PR.

Otherwise, FOG ON, YOU CRAZY DIAMOND.

File: app/views/homes/show.html.erb (hunk 2 above)
Comment (adarsh) [inline app/views/homes/show.html.erb]: Or maybe, "Recommend a TV Show here..."

File: app/assets/stylesheets/components/_header.scss (hunk 1 above)
Comment (gabebw) [inline app/assets/stylesheets/components/_header.scss]: Just turned on Hound. Variable'd: 8f16e13ba6c6a538e0984f1178104f66b7f9a6fc

File: app/views/homes/show.html.erb (hunk 2 above)
Comment (gabebw) [inline app/views/homes/show.html.erb]: > I feel like `The Wire` is a serious default. What about something more frivolous?

I learned it from talking to you, Adarsh! I LEARNED IT FROM TALKING TO YOU

> Also, this should be a constant, right?

Fixed both issues in 4b32c409ac09714d189c90c8c058eae8ed61a640

File: app/views/homes/show.html.erb (hunk 3 above)
Comment (gabebw) [inline app/views/homes/show.html.erb:21]: Good idea. I think it's out of scope for this PR -- want to make a card?

**Comment by gabebw** [comment]
> Is this responsive? 

Sort of. We should probably just hide "people to follow" on mobile:

youshouldtotally 2016-01-24 19-43-34

**Comment by gabebw** [comment]
Card for responsiveness: https://trello.com/c/jmugQqhQ/23-as-a-user-on-a-mobile-device-i-want-the-site-to-be-responsive

**Comment by adarsh** [comment]
You are a shining star.

## 

@adarshp http://www.twitter.com/adarshp
313.454.1515 (mobile)
See when I am free: http://bit.ly/adarshcal
